package.json

```diff
@@ -1,145 +1,25 @@
-    "pretty": "prettier --config .prettierrc \"scripts/**/*\" src/{__tests__,core,style,utils}/**/*{tsx,ts,jsx,js,css,scss} --write",
-    "lint": "eslint */**/*.{js,ts,tsx} --max-warnings=0",
-    "lint:fix": "yarn lint --fix && yarn lint:copyright --fix */**/*.{js,ts,tsx}",
-    "lint:copyright": "node scripts/copyrightLinter.js",
-    "copy-files": "cpx package.json lib && cpx \"{README,LICENSE,CHANGELOG}.md \" lib",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
-  },
-  "dependencies": {
-    "@itwin/itwinui-css": "^0.53.3",
-    "@itwin/itwinui-icons-react": "^1.5.0",
-    "@itwin/itwinui-illustrations-react": "^1.0.1",
-    "@tippyjs/react": "^4.2.5",
-    "@types/react-table": "^7.0.18",
-    "classnames": "^2.2.6",
-    "react-table": "^7.1.0",
-    "react-transition-group": "^4.1.1"
+    "build": "turbo run build",
+    "dev": "turbo run dev",
+    "lint": "turbo run lint",
+    "format": "turbo run format",
+    "test": "turbo run test"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.10",
-    "@storybook/addon-a11y": "^6.4.19",
-    "@storybook/addon-essentials": "~6.4.19",
-    "@storybook/addon-storysource": "~6.4.19",
-    "@storybook/addons": "~6.4.19",
-    "@storybook/api": "~6.4.19",
-    "@storybook/builder-webpack5": "~6.4.19",
-    "@storybook/components": "~6.4.19",
-    "@storybook/core-events": "~6.4.19",
-    "@storybook/manager-webpack5": "~6.4.19",
-    "@storybook/preset-typescript": "^3.0.0",
-    "@storybook/react": "~6.4.19",
-    "@storybook/theming": "~6.4.19",
-    "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^12.0.0",
-    "@testing-library/react-hooks": "^7.0.1",
-    "@testing-library/user-event": "^13.5.0",
-    "@types/classnames": "^2.2.7",
-    "@types/jest": "^27.0.0",
-    "@types/node": "^10.12.2",
-    "@types/react": "^17.0.3",
-    "@types/react-dom": "^17.0.3",
-    "@types/react-transition-group": "^2.9.2",
-    "@typescript-eslint/eslint-plugin": "^4.31.1",
-    "@typescript-eslint/parser": "^4.31.1",
-    "babel-loader": "^8.2.2",
-    "concurrently": "^5.3.0",
-    "cpx": "^1.5.0",
-    "creevey": "^0.8.0-beta.0",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-react": "^7.25.1",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "eslint-plugin-storybook": "^0.5.7",
-    "fast-glob": "^3.2.5",
-    "husky": "^4.0.0",
-    "inquirer": "^6.2.2",
-    "jest": "^27.0.6",
-    "jest-cli": "^27.0.6",
-    "jest-junit": "^12.0.0",
-    "lint-staged": "^12.1.2",
-    "markdown-to-jsx": "6.11.4",
-    "mkdirp": "^1.0.4",
-    "prettier": "2.2.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "rimraf": "^2.6.2",
-    "storybook-dark-mode": "^1.0.9",
-    "tippy.js": "^6.3.1",
-    "ts-jest": "^27.0.4",
-    "ts-loader": "^8.0.1",
-    "ts-node": "^8.0.2",
-    "typescript": "^4.4.3",
+    "turbo": "latest",
     "webpack": "5"
```

Comment:
Why it is still needed?

Reply:
It's in the PR description
<img width="774" alt="image" src="https://user-images.githubusercontent.com/9084735/162758270-6bf5540b-b928-4ab8-bacc-2cb4633f1af4.png">